webium/base_page.py

```diff
@@ -21,9 +21,15 @@ def _get_driver():
     _get_driver().implicitly_wait(timeout)
     try:
         def is_displayed():
-            element = getattr(self, element_name, None)
-            if not element:
+            try:
+                element = getattr(self, element_name)
+            except AttributeError:
                 raise WebiumException('No element "{0}" within container {1}'.format(element_name, self))
+            if isinstance(element, list):
+                if element:
```

Comment:
Semantically, it might be more clear if I wrote this as `if element != []:` instead of `if element:`. I can change it to that if you prefer. This is mostly my habit when checking for empty sequences/iterables.